param/__init__.py

```diff
@@ -563,7 +563,7 @@ class Number(Dynamic):
     allows, for instance, a GUI to know what values to display on
     sliders for the Number.
 
-    Example of creating a Number::
+    Example of creating a Number:
```

Comment:
@jlstevens, @ceball, @philippjfr, does anyone know if the :: syntax is still required here?  I don't think that was a typo; it was originally for a docstring interpreted as reStructuredText, where :: introduces a code example. But are our docstrings currently being interpreted as rst, or is this a holdover from some long-ago doc building system?

Reply:
Personally I know nothing about using :: so it looks like a typo to me...

Reply:
It's definitely [valid rst](http://docutils.sourceforge.net/docs/user/rst/quickref.html#literal-blocks); I just don't know if any of our tools are interpreting it as rst. Hopefully @ceball knows that.

Reply:
Ok, from discussing with @ceball it's clear that at least some tools are indeed able to process the rst, so please back out this one-character change.